### DISCUSSION TITLE
Support license from environment variable

### DISCUSSION THREAD

**Discussion by svrooij** [danielpalme/ReportGenerator#674] [Ideas]
It would be really nice if the license would be loaded from the environment variable `REPORTGENERATOR_LICENSE` then we don't have to include it in the command line each time.

**Comment by danielpalme** [comment]
@svrooij 
No problem. I just made the necessary changes in  2b62fbcfeabfe75b1c5d0224987f8fcb23adf87b.
The next release will contain this change.

Until then you can also create a txt file named `.netconfig` with the following content:

```
[ReportGenerator]
  license = YOUR_LICENSE
```

The file can be placed in your working directory or any parent folder.  
See also: https://github.com/danielpalme/ReportGenerator/?tab=readme-ov-file#netconfig-support

**Reply by svrooij** [reply]
Wait, I have this great idea and you have it implemented in +/-45 minutes?
Great work!!

Which is also why I asked my company to buy a license 😉

**Reply by danielpalme** [reply]
Thank you for buying a license!

**Reply by danielpalme** [reply]
New release 5.3.5 is now available!

**Reply by svrooij** [reply]
I already change the [workflow](https://github.com/svrooij/titanium-web-proxy/actions/runs/9319845945/workflow#L53-L57) so running it again and seeing what happens.

Yes it works perfectly! I'm also changing the output a little bit here so make it match the rest of the summary
https://github.com/svrooij/titanium-web-proxy/actions/runs/9337329962#summary-25699069545

**Comment by JohnVerheij** [comment]
This works perfectly
